rxbinding/src/androidTest/java/com/jakewharton/rxbinding/widget/RxSeekBarTest.java

```diff
@@ -51,13 +54,81 @@
     instrumentation.waitForIdleSync();
     assertThat(o.takeNext()).isEqualTo(0);
 
+    seekBar.setProgress(85);
```

Comment:
I am really surprised this works. You can't call this method directly since the test is running on a background thread. All these method calls need to be changed to:
```java
instrumentation.runOnMainSync(new Runnable() {
  @Override public void run() {
    seekBar.setProgress(/* whatever */);
  }
}
```